node/src/accountant/db_big_integer/big_int_db_processor.rs

```diff
@@ -50,19 +64,22 @@ impl<'a, T: TableNameDAO> BigIntDbProcessor<T> {
     }
 }
 
-impl<T: TableNameDAO + 'static> Default for BigIntDbProcessor<T> {
-    fn default() -> BigIntDbProcessor<T> {
+impl<T: TableNameDAO + 'static> Default for BigIntDatabaseProcessorReal<T> {
```

Comment:
Also, I think, we can definitely use `db` for database here. It's a well known short form, so it'll be useful here.

`BigIntDatabaseProcessorReal` -> `BigIntDbProcessorReal`
`BigIntDatabaseProcessor` -> `BigIntDbProcessor`